templates/web/base/reports/index.html

```diff
@@ -109,6 +109,22 @@ <h2 class="dashboard-subheading">[% loc('Top 5 responsive councils') %]</h2>
         </table>
       [% END %]
     </div>
+    <div class="dashboard-item dashboard-item--6">
```

Comment:
We don't want anther column, we want this inside the above `div` as it's the contents of that column we're changing, not an extra column.